db/schema.rb

```diff
@@ -235,6 +237,7 @@
     t.boolean "send_notification_email_completed"
     t.datetime "originally_submitted_at"
     t.index ["core_search_fields"], name: "index_prior_authority_applications_on_core_search_fields", using: :gin
+    t.index ["created_at", "service_type"], name: "idx_pas_service_type_created_at", where: "((service_type IS NOT NULL) AND ((service_type)::text <> ''::text))"
```

Comment:
has that crept in from your local DB schema?

Reply:
Good spot, it's from another PR that is on hold